src/react/components/searchNoResults.js

```diff
@@ -0,0 +1,61 @@
+import HeroIcon from "./heroIcon";
+import React from "react";
+
+const columns = [
+  {
+    heading: 'Popular Topics',
+    links: [
+      {title: 'Reunion 2021', url: 'https://reunion.stanford.edu/'},
+      {title: 'Programs & Resources for Students', url: 'https://students.alumni.stanford.edu/'},
+      {title: 'Request your official Stanford transcript', url: 'https://registrar.stanford.edu/students/transcripts/transcripts-alumni'},
+      {title: 'Stanford Alumni Email', url: 'https://alumni-preview.stanford.edu/perks/email/'},
```

Comment:
Everything `https://alumni-preview.stanford.edu` should be changed to `https://alumni.stanford.edu`